public/components/custom_panels/helpers/utils.tsx

```diff
@@ -58,11 +59,23 @@ export const convertDateTime = (datetime: string, isStart = true, formatted = tr
   } else {
     returnTime = dateMath.parse(datetime, { roundUp: true });
   }
-
   if (formatted) return returnTime!.utc().format(PPL_DATE_FORMAT);
   return returnTime;
 };
 
+export const convertDateTimeToEpoch = (datetime: string, isStart = true, formatted = true) => {
```

Comment:
Can we re-use or merge `convertDateTime` function here?

Reply:
I can take a look into this